packages/resource-deployment/scripts/deploy-rest-api.sh

```diff
@@ -47,23 +47,25 @@ fi
 
 echo "Starting deployment for REST api..."
 
-echo "Deploying named values"
-deployResource "$apiTemplates/model-namedValues.template.json"
+# Commenting out empty template files, since each action adds 30-40 secs of deployment time.
+# We may need to have a single ARM template to improve performance.
+# echo "Deploying named values"
+# deployResource "$apiTemplates/model-namedValues.template.json"
 
-echo "Deploying api version sets"
-deployResource "$apiTemplates/model-apiVersionSets.template.json"
+# echo "Deploying api version sets"
+# deployResource "$apiTemplates/model-apiVersionSets.template.json"
 
 echo "Deploying products"
 deployResource "$apiTemplates/model-products.template.json"
 
-echo "Deploying Loggers"
-deployResource "$apiTemplates/model-loggers.template.json"
+# echo "Deploying Loggers"
+# deployResource "$apiTemplates/model-loggers.template.json"
 
 echo "Deploying backends"
 deployResourceWithFunctionName "$apiTemplates/model-backends.template.json"
 
-echo "Deploying authorization servers"
-deployResource "$apiTemplates/model-authorizationServers.template.json"
+# echo "Deploying authorization servers"
+# deployResource "$apiTemplates/model-authorizationServers.template.json"
```

Comment:
we have repo history to see old version, if we do not need this we should remove commented out code